src/main/java/com/exasol/adapter/installer/dialect/Db2DialectProfile.java

```diff
@@ -0,0 +1,57 @@
+package com.exasol.adapter.installer.dialect;
+
+import com.exasol.adapter.installer.UserInput;
+
+/**
+ * A DB2 dialect profile.
+ */
+public class Db2DialectProfile extends AbstractVirtualSchemaProfile {
```

Comment:
That's quite a lot of duplicate code here (see sonar analysis). 
I'm wondering if you really need to do this as java classes. Would it be possible to do this using configuration? E.g. in yml. Files?
One advantage would be, that we could place these files in the virtual schema repositories. That would reduce coupling between the virtual schemas and this repo.

Reply:
I remember we discussed this before starting this repository, I think this installer has its own lifecycle and we don't want to couple it with other repositories. Maybe this can be re-considered in the future, but this prototype will have dialects inside, I think. 
I don't see any advantages in using a duplicated configuration files over java classes. Java classes instead require implementation and you can miss something in a plain text file.